passkey-types/src/webauthn/attestation.rs

```diff
@@ -569,12 +787,59 @@ pub enum ClientDataType {
     /// Serializes to the string `"webauthn.get"`
     #[serde(rename = "webauthn.get")]
     Get,
+
+    /// Serializes to the string `"payment.get"`
+    /// This variant is part of the Secure Payment Confirmation specification
+    ///
+    /// See <https://www.w3.org/TR/secure-payment-confirmation/#client-extension-processing-authentication>
+    #[serde(rename = "payment.get")]
+    PaymentGet,
+}
+
+impl fmt::Display for ClientDataType {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let renamed = serde_json::to_string(self).unwrap();
+        write!(f, "{}", renamed.trim_matches('"'))
+    }
 }
 
 #[cfg(test)]
 mod tests {
-
     use super::CredentialCreationOptions;
+    use crate::webauthn::{
+        collected_client_data_to_json_bytes, ClientDataType, CollectedClientData,
+    };
+
+    // Normal client data from Chrome assertion
+    const CLIENT_DATA_JSON_STRING: &str = r#"{
+        "type":"webauthn.get",
+        "challenge":"ZEvMflZDcwQJmarInnYi88px-6HZcv2Uoxw7-_JOOTg",
+        "origin":"http://localhost:4000",
+        "crossOrigin":false
+    }"#;
```

Comment:
**Suggestion:** Continuing the [maintenance point from here](https://github.com/1Password/passkey-rs/pull/6#discussion_r1414559123). We can have a second copy of this CCD with `crossOrigin:true` then test the 3 cases. 
* `None`
* `Some(false)`
* `Some(true)`

Then we get the benefit of validating that the serialization is done the same way as the spec without needing the manual serializer.